static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -60,8 +60,10 @@ function UserActionsNavigator({event, crumbs}: Props) {
 
   const {startTimestamp} = event || {};
   const userActionCrumbs =
-    crumbs && transformCrumbs(crumbs).filter(crumb => USER_ACTIONS.includes(crumb.type));
-  const isLoaded = userActionCrumbs && startTimestamp;
+    (crumbs &&
+      transformCrumbs(crumbs).filter(crumb => USER_ACTIONS.includes(crumb.type))) ||
+    [];
```

Comment:
Use a ternary`a ? b : c` instead of `a && b || c`. 

Or my style would be to default the crumbs variable to `[]`
```suggestion
    transformCrumbs(crumbs || []).filter(crumb => USER_ACTIONS.includes(crumb.type)));
```